cookiecutter-python-library/{{cookiecutter.library_name}}/tests/test_{{cookiecutter.library_name}}.py

```diff
@@ -1,3 +1,11 @@
 """
 Tests for {{cookiecutter.library_name}}.py.
 """
+
+
+def test_placeholder():
+    """
+    Placeholder to allow pytest to succeed before real tests are in place.
+
+    (If there are no tests, it will exit with code 5.)
```

Comment:
May be silly, but I wonder if a comment like the following will stand-out in terms of cleaning up:
```
TODO: Delete this test once there are real tests.
```

Reply:
Hmm... I could also make the test be skipped, which might get people's attention when they run tests:

```
@pytest.mark.skip(reason="Placeholder to allow pytest to succeed before real tests are in place.")
def test_placeholder():
    """
    TODO: Delete this test once there are real tests.
    """
```

Or, for even more fun, the test could fail with a message ("delete me") unless a specific environment variable is set, which would allow us to run cookiecutter tests that pass. :-)